dimension-reduction/results/pca_scores.tsv

```diff
@@ -1,256 +1,256 @@
 refinebio_accession_code	PC1	PC2	histology	subgroup
-GSM917142	-37.545636158651675	-142.36326956107607	Classic	Group 4
```

Comment:
I am not sure myself, was thinking that it may have to do with some update to the input files but I'll take a deeper look into it to see if I could pinpoint the source of the changes.